addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -491,6 +492,13 @@ ListRenderer.include({
      * @override
      */
     updateState: function (state, params) {
+        this.oldColumns = this.columns;
+        this.columnWdiths = false;
```

Comment:
hmm, read carefully this line... Maybe it is useless, maybe a test is missing...